ansible/roles/collect_sos_report/tasks/main.yml

```diff
@@ -0,0 +1,43 @@
+---
+- name: Create temporary directory for sos report
+  ansible.builtin.tempfile:
+    path: /var/tmp
+    prefix: sosreport-
+    state: directory
+  become: true
+  register: r_sos_tempdir
+
+- name: Collect sosreport
+  ansible.builtin.command:
+    cmd: >-
+      sos report --batch --tmp-dir {{ r_sos_tempdir.path }}
+  become: true
+  ignore_errors: true
```

Comment:
should we report or simply ignore?

Reply:
The ignore_errors will still show an error in the logs. Not sure what else there would be to report?

Reply:
Ex:
```
TASK [collect_sos_report : Collect sosreport] ************************************************************************************************************************************************************************************************
Monday 24 March 2025  13:13:03 -0400 (0:00:02.104)       0:00:02.142 ********** 
fatal: [bastion]: FAILED! => {"changed": false, "cmd": "sos report --batch --tmp-dir /var/tmp/sosreport-udn85gy3", "msg": "[Errno 2] No such file or directory: b'sos'", "rc": 2, "stderr": "", "stderr_lines": [], "stdout": "", "stdout_lines": []}
...ignoring
```